wundergraph/.wundergraph/wundergraph.config.ts

```diff
@@ -0,0 +1,38 @@
+import { configureWunderGraphApplication, cors, EnvironmentVariable, introspect, templates } from '@wundergraph/sdk';
+import server from './wundergraph.server';
+import operations from './wundergraph.operations';
+
+const ecosystem = introspect.graphql({
+	apiNamespace: 'ecosystem',
+	url: process.env.ECOSYSTEM_URL || 'https://countries.trevorblades.com/graphql',
```

Comment:
Please do not hardcode example values into our codebase, but provide a clear instruction in the readme on how to try it out. If the environment variables are not provided, the service must fail